101-perforce-helix-core-server/azuredeploy.json

```diff
@@ -175,15 +183,16 @@
   },
   "resources": [
     {
-      "apiVersion": "2019-05-01",
-      "name": "pid-6c922c23-cc65-4b76-8290-74feb0f3c476",
+      "apiVersion": "2019-10-01",
```

Comment:
this is for marketplace publisher identification, so just remove it from the sample

Reply:
OK - removed the resource.